src/main/rx/msp.c

```diff
@@ -59,7 +59,7 @@ void rxMspFrameReceive(uint16_t *frame, int channelCount)
     rxMspFrameDone = true;
 }
 
-static uint8_t rxMspFrameStatus(rxRuntimeState_t *rxRuntimeState)
+uint8_t rxMspFrameStatus(rxRuntimeState_t *rxRuntimeState)
```

Comment:
From code, it look that rxMspFrameStatus is function with side effect- it will reset `rxMspFrameDone`. Calling in from wrong place (= ignoring RX_FRAME_COMPLETE) will probably confuse RxMSP state automaton

Reply:
Thanks @ledvinap. What would you suggest as a better solution?

Reply:
@ledvinap I can potentially add a timestamp here `rxMspReceivedAt` which would save when the last MSP package was received and then check it to see if there was a new package since last last processing inside of `rxFrameCheck`. But that would be kind of duplicating this functionality.

Reply:
Sorry, I did not study code too closely. It is true that rxMspFrameDone is reset, but rx0owerride is typically single consumer, so it wont break anything (overriding msp with msp may not work). And MSP receiver is almost stateless.

`rxMspOverrideFrameStatus` is a bit cleaner.

Maybe put  
```
if (IS_RC_MODE_ACTIVE(BOXMSPOVERRIDE) && rxConfig()->msp_override_channels_mask) {
        if (rxMspOverrideFrameStatus() & RX_FRAME_COMPLETE) 
            signalReceived = true;
```

before https://github.com/betaflight/betaflight/blob/99ba203646ebc6881785033da42348dd19470521/src/main/rx/rx.c#L550C4-L550C26

It will reduce code duplication. And processing rc both on receiver and msp update is necessary